config.go

```diff
@@ -314,6 +319,14 @@ func (cfg *Config) validate() error {
 		}
 	}
 
+	if cfg.Interval < minIntervalValue {
+		return fmt.Errorf("interval value must be >= %.1f", minIntervalValue)
```

Comment:
For usability's sake, how about just force set the minimum allowed value and print a warning to the log?

Reply:
I think this isn't a good idea because most users don't have 'warning' log level enabled and they are not looking into log files in general. Accordingly, they will never know that there are such restrictions. Also, explicit is better than implicit. 
Maybe @thorstenkramm do have some thoughts on this, too?

Reply:
I didn't exactly mean log using the warning log-level, but just to add a line to the log about that we use default values because given value is out of range.

In multiple scenarios the user will not see the returned error anywhere and will be forced to consult log files with your proposed implementation just to figure out what went wrong.

For a end user I'm not sure I would agree hard failure on such trivial condition is good user experience, but I have no strong feelings either way.